eslint.config.mjs

```diff
@@ -0,0 +1,85 @@
+// @ts-check
```

Comment:
Maybe we should live-review this file? Let me know if you think that would be helpful

Reply:
I don't think it is necessary. But happy to discuss if that helps.
<!-- Reviewable comment -OBQon-wAURA6An_o7Qw:b-1oks5q -->
<!-- Sent from Reviewable.io -->